orcid-web/src/main/resources/freemarker/manage.ftl

```diff
@@ -249,39 +243,6 @@
                         </div>
                     </td>
                 </tr>
-                <#if !profile.orcidInternal.preferences.notificationsEnabled>
```

Comment:
Yea, it references OrcidProfile and everyone now has notifications enabled,
so it's not needed.

On Thu, Jun 23, 2016 at 12:42 PM, Angel Montenegro <notifications@github.com
> wrote:

> In orcid-web/src/main/resources/freemarker/manage.ftl
> <https://github.com/ORCID/ORCID-Source/pull/2949#discussion_r68301530>:
>
> > @@ -249,39 +243,6 @@
> >                          </div>
> >                      </td>
> >                  </tr>
> > -                <#if !profile.orcidInternal.preferences.notificationsEnabled>
>
> Hey @rcpeters <https://github.com/rcpeters> not sure why you removed
> this? is this something we dont want anymore?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ORCID/ORCID-Source/pull/2949/files/8a9a8a814edc200a3e55f3ab13d679a1f4aef374#r68301530>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALx5LbpXNBtXgfC6TKvla0_qGRjC492ks5qOuHCgaJpZM4I9KMj>
> .
>



-- 
Robert Peters
info@rcpeters.com
805.440.9056